### ISSUE TITLE
Error:Could not get unknown property 'prepareReleaseDependencies' for project ':someLib' of type org.gradle.api.Project.

### ISSUE THREAD

**Issue by LSDsl** [adwiv/android-fat-aar#84]
Error:Could not get unknown property 'prepareReleaseDependencies' for project ':someLib' of type org.gradle.api.Project.

What is wrong?

**Comment by TTKatrina** [comment]
Use "compile" or "provided" instead of "embeded" may resovle your problem.

**Comment by LSDsl** [comment]
Ha-ha.. I use this lib to embeded aar. Not compile or provided )

P.S. Fix this by downgrade from AS 3 to AS 2.3.3

**Comment by TTKatrina** [comment]
here is the same question: https://github.com/adwiv/android-fat-aar/issues/45

**Comment by AvivVegh** [comment]
Still doesn't work on gradle 3.0, is it work to someone ?

**Comment by Soloer** [comment]
can't work on gradle 3.0?

**Comment by zhpanvip** [comment]
same error with you

**Comment by mfdeveloper** [comment]
I've "fixed" this downgrading gradle to `3.3` version on `gradle-wrapper.properties` file, and change the `dependencies` on `build.gradle` file, like the code snippet below:

```groovy
dependencies {
    classpath 'com.android.tools.build:gradle:2.3.3'
}
```

**Comment by kolyneh** [comment]
Hi, I've "fixed" this downgrading gradle to 2.14.1 version. 

gradle-wrapper.properties :
distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip

build.gradle
dependencies {
    classpath 'com.android.tools.build:gradle:2.3.3'
}

**Comment by JasonToJan** [comment]
Don't want to downgrade the gradle.. Could somebody help me?

**Comment by yihongyuelan** [comment]
> Hi, I've "fixed" this downgrading gradle to 2.14.1 version.
> 
> gradle-wrapper.properties :
> distributionUrl=https://services.gradle.org/distributions/gradle-2.14.1-all.zip
> 
> build.gradle
> dependencies {
> classpath 'com.android.tools.build:gradle:2.3.3'
> }

You are right, I'v tried to fix fat-aar error code and compile with new gradle version 4.x.
Opps, Failed again and again, so I create a simple AndroidStudio project, just merge all FUCKING aar files with lower gradle version. 

https://github.com/yihongyuelan/android-MergeAAR
